B827EBFFFE89386B.json

```diff
@@ -0,0 +1,18 @@
+{ 
+	"gateway_conf": { 
+		"gateway_ID": "B827EBFFFE89386B", 
+		"servers": [ 
+			{ 
+				"server_address": "eu.thethings.network", 
```

Comment:
I think you mean `router.eu.thethings.network`,  instead, right? Could you please update?